stbt-batch.d/report

```diff
@@ -12,6 +12,8 @@
 #/ Generates a detailed report in <testrun directory>/index.html
 #/ and a summary report in <testrun directory>/../index.html.
 
+IFS=$'\n'
+
```

Comment:
Ok, I guess so.  I don't particularly like that this will change the behavior of the whole script, when it's only needed for a bit below.  I wouldn't say I'm a confident enough bash-scripter to say whether this is idiomatic or not.

An alternative to:

    for resultsdir in $(
      for rundir in "$@"; do dirname "$(abspath "$rundir")"; done | sort | uniq)
    do

could be:

    for rundir in "$@"; do dirname "$(abspath "$rundir")"; done | sort | uniq | while read resultsdir
    do

None of this is perfect though.  bash is such a pain for things if you want them to be right in all circumstances.